Atomic/backends/_docker.py

```diff
@@ -71,7 +71,10 @@ def has_image(self, img):
                              "matching '{0}'. They are:\n    {1} \n{2}"
                              .format(img, "\n    ".join(repo_tags),
                                      err_append))
-        return self._make_image(img, self._inspect_image(img), deep=True)
+
+        img = name_search[0]['RepoTags'][0]
+        inspected_image = self.inspect_image(image=img)
+        return inspected_image
```

Comment:
I'm OK with this approach but have you checked if we rely on an error being raised if nothing is found? My guts tell me we do.